src/router/HomeView/components/DonationDialog/DonationDialog.tsx

```
@@ -0,0 +1,147 @@
+						fullWidth
+						aria-label={formatMessage({ id: 'donation_dialog.submit_button.label' })}
+						onClick={handleSubmit((data) => {
+							const submitData = {
+								monthlyAmount: parseCurrencyToNumber(data.value),
+								lastMonth: format(month, 'MMMM yyyy'),
+								currency: OCurrencyName[currency as keyof typeof OCurrencyName],
+							};
+
+							console.log(submitData);
+
+							handleClose && handleClose();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)